packages/arb-token-bridge-ui/src/util/orbitChainsData.json

```diff
@@ -964,48 +964,6 @@
         }
       }
     },
-    {
```

Comment:
Needed to remove L3X Testnet because it's unreachable and breaking our Orbit json validator script.